src/app/modules/item/components/item-remove-button/item-remove-button.component.scss

```diff
@@ -6,10 +6,6 @@
   align-items: center;
 }
 
-.spinner {
-  margin-right: .4rem;
-}
-
```

Comment:
It's weird that this styling has become useless Oo
Is it the case for group-remove button as well ?

Reply:
New loading component already has padding. Also just pushed for group-remove the same fix